src/components/DatePicker/DatePicker.tsx

```diff
@@ -288,6 +299,8 @@ export const DatePicker = chakra(
     const onChangeDefault = (date: Date, value: string) => {
       setFullDate({ ...fullDate, [value]: date });
       onChange && onChange({ ...fullDate, [value]: date });
+
+      inputRefs[value].current.focus();
```

Comment:
This works but should the created ref be focused or the merged ref? If you did something like `mergedStartDateRefs.current.focus()`, does that work? If so, perhaps `inputRefs.startDate` should be initialized to `useMergeRefs(startDateInputRef, ref)` so that is focused instead.

Reply:
Is this what you mean? And then using inputRefs.startDate or inputRefs.endDate throughout? If so, I wasn't able to get this to work.

``` const inputRefs = {
      startDate: useMergeRefs(startDateInputRef, ref),
      endDate: useMergeRefs(endDateInputRef, refTo),
    };
```

Reply:
It was what I meant. Okay not a big deal, thanks for trying it!